### PR TITLE
Update AREA V1.0 format description

### PR DESCRIPTION
**Pull request by Argent77**

### CODE REVIEW

File: file_formats/ie_formats/are_v1.htm [hunk 1]
```diff
@@ -97,24 +97,48 @@
...
+              <li>bit 6: Travel not allowed</li>
+            </ul>
+
             PST:
             <ul>
-              <li>0 - Rest allowed</li>
-              <li>2 - "You cannot rest here."</li>
-              <li>4 - "You cannot rest right now."</li>
-              <li>6 - "You must obtain permission to rest here."</li>
+              <li>bit 0: Save not allowed</li>
```
Comment (lynxlynxlynx) [inline file_formats/ie_formats/are_v1.htm]: Are you sure? I don't this bit is used in pst.

Comment (Argent77) [reply file_formats/ie_formats/are_v1.htm]: The bit seems to work as described in PST. I have just tested it to make sure.

File: file_formats/ie_formats/are_v1.htm [hunk 2]
```diff
@@ -1924,8 +2031,13 @@
...
             <td>0x0048</td>
-            <td>4 (dword)</td>
-            <td>Unused</td>
+            <td>2 (word)</td>
+            <td>Animation width (BGEE)</td>
+          </tr>
+          <tr>
+            <td>0x004a</td>
+            <td>2 (word)</td>
+            <td>Animation height (BGEE)</td>
```
Comment (lynxlynxlynx) [inline file_formats/ie_formats/are_v1.htm]: Can you note that these two are unused in the other engines?

Comment (Argent77) [reply file_formats/ie_formats/are_v1.htm]: Looks like these values are only needed when WBM or raw PVRZ resources are used for background animations. For regular BAM animations these values are ignored. I'll update the format description accordingly. (Looks like I forgot to add EE-specific flags as well.)

File: file_formats/ie_formats/are_v1.htm (hunk 1 above)
Comment (lynxlynxlynx) [reply file_formats/ie_formats/are_v1.htm]: ok, good to know. Our previous research is wrong then - maybe it hinged on the old iesdp description.

**Pull request merged**